roles/vald/defaults/main.yml

```diff
@@ -1,16 +1,18 @@
 ---
 # Maintains directories and paths
-vald_home_dir: /opt/{{ service_identifier }}
+vald_home_dir: "{{ vald_axelar_home }}"
```

Comment:
Still use /opt/{{ service_identifier }} for vald_home_dir

Reply:
I will explain it in the call

Reply:
So go with seperate home directory for vald ?

Reply:
yes please